lib/client-assets.php

```diff
@@ -269,6 +269,11 @@ function gutenberg_register_scripts_and_styles() {
 		filemtime( gutenberg_dir_path() . 'edit-post/build/index.js' ),
 		true
 	);
+	wp_add_inline_script(
+		'wp-edit-post',
+		gutenberg_get_script_polyfill( array( 'false' => 'formdata' ) ),
```

Comment:
I may have been confused previously. I assumed "`false` for forced" was just for debugging purposes. Can't we have a real condition to check here for whether FormData is supported? As-is, this isn't really any different than the original implementation, just loaded as a separate resource.

Reply:
If I understand correctly we can't, because the check is more complex, depending on the browser somethings could be supported and others not.

Reply:
looking at the code of the polyfill for confirmation.

Reply:
I know there are some odd quirks with FormData and Safari (something about `entries`), so you may be right, but if we can, it would be good to identify.

Reply:
Oh it looks like they are doing this check `!window.FormData || !window.FormData.prototype.keys` We can probably do it here as well. Trying.